api/common.go

```diff
@@ -3,7 +3,7 @@ package api // import "github.com/docker/docker/api"
 // Common constants for daemon and client.
 const (
 	// DefaultVersion of the current REST API.
-	DefaultVersion = "1.46"
+	DefaultVersion = "1.47"
```

Comment:
APi 1.46 has not yet been released (but will be with docker v27.0.0) so you can keep it at the current version.